content/casestudy/hellofresh.md

```diff
@@ -0,0 +1,25 @@
+# HelloFresh logging platform serving multiple countries.
+
+TBD
```

Comment:
Do you have a plan to update this TBD?

Reply:
hi @repeatedly  i am working with my communication manager. she will give it a short summary in next few days. 
Thanks

Reply:
I updated